script.js

```diff
@@ -1,66 +1,462 @@
+		if (list.find(elem => elem.id.toString() === targetElem).editMode === false) {
+			if (_('check-' + targetElem).checked === true) {
+				_('desc-' + targetElem).classList.remove("checkboxChecked");
+			}
+			list.find(elem => elem.id.toString() === targetElem).editMode = true;
+			_('desc-' + targetElem).contentEditable = true;
+			_('check-' + targetElem).setAttribute("disabled", "disabled");
+			editimage.classList.add("saveButtonImage");
+			editimage.classList.remove("editButtonImage");
+			editimage.setAttribute("src", './images/save.png')
+			_('desc-' + targetElem).focus();
+			let placeOfEdit = _('desc-' + targetElem)
+			function placeCaretAtEnd(el) {
+				el.focus();
+				if (typeof window.getSelection != "undefined"
+					&& typeof document.createRange != "undefined") {
+					var range = document.createRange();
+					range.selectNodeContents(el);
+					range.collapse(false);
+					var sel = window.getSelection();
+					sel.removeAllRanges();
+					sel.addRange(range);
+				} else if (typeof document.body.createTextRange != "undefined") {
+					var textRange = document.body.createTextRange();
+					textRange.moveToElementText(el);
+					textRange.collapse(false);
+					textRange.select();
+				}
+			}
+
+			placeCaretAtEnd(placeOfEdit);
+		}
+		else {
+			if (_('check-' + targetElem).checked === true) {
+				_('desc-' + targetElem).classList.add("checkboxChecked");
+			}
+			list.find(elem => elem.id.toString() === targetElem).description = _('desc-' + targetElem).innerText;
+			list.find(elem => elem.id.toString() === targetElem).editMode = false;
+			_('check-' + targetElem).removeAttribute("disabled");
+			_('desc-' + targetElem).contentEditable = false;
+			editimage.classList.remove("saveButtonImage");
+			editimage.classList.add("editButtonImage");
+			editimage.setAttribute("src", './images/edit.png')
+			storeLocal();
+			setTimeout(() => { _("input_notes").focus() }, 200)
+		}
+	}
+	editItem.addEventListener("click", editContent)
+	editimage.addEventListener("click", editContent)
+	description.addEventListener("dblclick", editContent)
+	description.addEventListener("keydown", (e) => {
+		e.stopImmediatePropagation();
+		if (e.code === "Enter") {
+			editContent(e);
+		}
+	})
+
+	//Remove button event
+	let removedItem = _("remove-" + list[list.length - 1].id);
+	let removeimage = _("removeimage" + list[list.length - 1].id);
+	function showDeletionModal(e) {
+		e.stopPropagation();
+		_("input_notes").blur();
+		_('removeModal').classList.add("emptyModalParent");
+		let id = e.target.id.replace("removeimage", "").replace("remove-", "");
+		_('confirm').setAttribute("data-id", id)
+	}
+	removedItem.addEventListener("click", showDeletionModal);
+	removeimage.addEventListener("click", showDeletionModal);
+
+	storeLocal();
+
+} //end of additem function
+
+
+
+function deleteItem(e) {
+	_("item" + e.target.dataset.id).remove();
+	_('removeModal').classList.remove("emptyModalParent");
+	_("input_notes").focus();
```

Comment:
We're using `_("input_notes")` a lot in the application. Instead of fetching it each time, declare a global variable and fetch it once from the DOM, and re-use it every time later on.

 `document.getElementById` is an expensive task, and overusing it will cause performance issues. So do this for all the elements that are fetched from DOM somewhat regularly.